src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -29,19 +29,30 @@ import scala.util.Try
 case class GooglePriceList(prices: GooglePrices, version: String, updated: String)
 
 /** Partial price list. Attributes can be added as needed to import prices for more products. */
-case class GooglePrices(cpBigstoreStorage: UsPriceItem)
+case class GooglePrices(cpBigstoreStorage: UsPriceItem,
+                        cpComputeengineInternetEgressNA: UsTieredPriceItem)
 
 /** Price item containing only US currency. */
 case class UsPriceItem(us: BigDecimal)
 
+/** Tiered price item containing only US currency. */
```

Comment:
Can you document what the map means? It looks to me like size to cost, but I'm not sure what the size unit is or how to interpret it.

Note to self: I need to document `UsPriceItem` to say that it's the price in dollars/GB.